ui-components/src/page_components/listing/ListingsList.tsx

```diff
@@ -51,7 +51,11 @@ const ListingsList = (props: ListingsProps) => {
           />
         </div>
         <div className="listings-row_content">
-          <h4 className="listings-row_title">{t("listings.waitlist.open")}</h4>
+          {listing.waitlistMaxSize !== null &&
+            listing.waitlistCurrentSize !== null &&
+            listing.waitlistCurrentSize < listing.waitlistMaxSize && (
+              <h4 className="listings-row_title">{t("listings.waitlist.open")}</h4>
+            )}
```

Comment:
I'm wondering if it would be useful to have a computed/virtual key on listings for this case, something like listing.showWaitlist.

If we wanted to do this, we could update the listing entity to use TypeOrm's AfterLoad decorator, and we would add it to listing.entity with something like:
```
// declare the computed field
showWaitlist?: boolean

// set computed values
@AfterLoad()
setComputed() {
  this.showWaitlist = this.waitListMaxSize !== null && this.waitlistCurrentSize !== null && this.waitlistCurrentSize < this.waitlistMaxSize
}
```

Reply:
Having not yet touched the backend, curious about @pbn4 's thoughts?

Reply:
Yes doable, do you want me to implement it backend side? Seems pretty straightfoward and I agree backend is a better place for such conditional logic.

Reply:
@emilyjablonski @seanmalbert

Reply:
I'll give this a first pass! Thanks 👍

Reply:
@emilyjablonski I don't think there is need for @AfterLoad, we could make it with such defining getter method on a property and @Expose that, checkout of how generating a slug property is handled because it's the same concept.

Reply:
That's a good point @pbn4 , it's the same affect, and not putting all of our eggs in TypeOrm's basket is probably a good idea.